src/platform_impl/macos/app_state.rs

```diff
@@ -457,3 +464,17 @@ unsafe fn window_activation_hack(ns_app: id) {
         }
     }
 }
+fn apply_activation_policy(app_delegate: &Object) {
+    unsafe {
+        use cocoa::appkit::NSApplicationActivationPolicy::*;
+        let ns_app = NSApp();
+        // We need to delay setting the activation policy and activating the app
+        // until we have the main menu all set up. Otherwise the menu won't be interactable.
```

Comment:
```suggestion
        // until `applicationDidFinishLaunching` has been called. Otherwise the
        // menu bar won't be interactable.
```

I mean, we call this before `menu::initialize`, so I think this new wording is more correct?

Reply:
You are absolutely right.